### ISSUE TITLE
Uninstall style_settings and then Install generates an php Error

### ISSUE THREAD

**Issue by vtad** [backdrop-contrib/style_settings#1]
ConfigStorageReadException: The configuration file "style_settings.settings" could not be read. in ConfigFileStorage->read() (line 898 of /home/www/backdrop/core/includes/config.inc).

**Comment by lolandese** [comment]
Good catch. Related to a core issue backdrop/backdrop-issues#511 but caused by a syntax error. In this case a trailing comma after the last JSON property that shouldn't be there.

Thanks for reporting.

P.S.:
It looks the error can leave some traces even after correcting it.  If you still get the error, restart the server with:
`sudo service apache2 restart`

**Comment by yorkshire-pudding** [comment]
@lolandese - please can you create a new release for this as the commit fixing this is not in the latest release.
